analysis/preregistration_template.Rmd

```diff
@@ -128,27 +138,34 @@ To be honest, I don't know how I might apply this to my work or any future works
 
 *Describe any data preparation and processing steps, including manipulation of environmental layers, e.g. standardisation and geographic projection.* 
 
-*Describe how you will separate and distinguish between raw data, manipulated data, and outputs from modelling or any analyses of the model.*
+*Describe how you will separate and distinguish between raw data, manipulated data, and outputs from modelling or any analyses of the model.* HF - Not sure exactly what this means
```

Comment:
```suggestion
```

Deleted! I think parsimony should rule over this one, since it's vague, confusing, and it might not work for all projects.